.editorconfig

```diff
@@ -0,0 +1,17 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+
+[*]
```

Comment:
Hi @tsingh2k15 , great work here! I suggest to add the this file only for UI code as the conventions for our js code and golang code are different in terms of spaces and indenting.

Reply:
> Hi @tsingh2k15 , great work here! I suggest to add the this file only for UI code as the conventions for our js code and golang code are different in terms of spaces and indenting.

@kushthedude Thanks for the feedback! I will work on the review comment, can you clarify my below questions?

- Initially I added this file under two UI sub repos under the main repo, but later I thought moving it to root level folder can be better. I can move it back to individual UI folders, but that would require it to be present under two sub repos right? There are two of them - "ui" and "provider-ui". Please share your thoughts if that is okay?
- Another reason I moved it to root directory was my thoughts on having just one file where we can include rules for *.go files and *.js files. Is this feasible to club the rules in one file, that may be easy to maintain if most of the rules are common across?

Reply:
I agree that we should have single EditorConfig file for ease of workflow & presentation of repository.

> * . Is this feasible to club the rules in one file, that may be easy to maintain if most of the rules are common across?

Even I think we should have the same indents & spaces over the repository. I will open the issue for it, I think only place to make change will be the ESLint rules file.